packages/graphql/src/api-v6/queryASTFactory/resolve-tree-parser/ResolveTreeParser.ts

```diff
@@ -180,6 +183,46 @@ abstract class ResolveTreeParser<T extends ConcreteEntity | Relationship> {
         const relationshipTreeParser = new RelationshipResolveTreeParser({ entity: relationship });
         return relationshipTreeParser.parseOperation(resolveTree);
     }
+
+    private parseConnectionArgs(resolveTreeArgs: { [str: string]: any }): GraphQLConnectionArgs {
```

Comment:
We need to start thinking on splitting the parser in different classes/functions